lib/licensed/command/version.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+module Licensed
+  module Command
+    class Version
+      def initialize
+      end
+
+      def run
+        $stdout << "#{Licensed::VERSION}\n"
```

Comment:
It's a bit of overkill for whats needed here, but could you use `Licensed::UI::Shell#info` to output the version?  That would keep output methods consistent throughout the program.  🙇

Reply:
Makes sense. I was a little back and forth between `$stdout` and `puts`. `Licensed::UI::Shell#info` answers that question. 😂